AdminWebsite/AdminWebsite.UnitTests/Controllers/WorkAllocationController/GetAllocationHearingsTests.cs

```diff
@@ -1,29 +1,29 @@
-﻿using FluentAssertions;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc;
-using Moq;
-using NUnit.Framework;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using BookingsApi.Client;
 using AdminWebsite.Contracts.Responses;
 using Autofac.Extras.Moq;
+using BookingsApi.Client;
 using BookingsApi.Contract.Requests;
 using BookingsApi.Contract.Responses;
+using FluentAssertions;
```

Comment:
Migrated work allocation endpoints to its own controller as discussed with @will-craig